### PR TITLE
[Sophia]Tasks 51 to 60

### PR DESCRIPTION
**Pull request by SofiMat**

### CODE REVIEW

File: tasks/59_sorted-sequence/sorted-sequence.js [hunk 1]
```diff
@@ -0,0 +1,17 @@
+export function uniteUnique(arr1, arr2, arr3) {
+  let finalArray = [];
+  for (let i = 0; i < arguments.length; i++) {
+  // for (let i = 0; i < (...args).length; i++) {//doesn't work
```
Comment (dverbovyi) [inline tasks/59_sorted-sequence/sorted-sequence.js]: don't commit the code which doesn't work, especially commented one

File: tasks/58_missed-letters/missed-letters.js [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+
+    if (secondCharacter - firstCharacter > 1) {
+      addCharacters(secondCharacter, firstCharacter);
+    }
+    firstCharacter = str.charCodeAt(i);
+  }
+  if (valueToReturn === '') {
+    return undefined;
+  } else {
+    return valueToReturn;
```
Comment (dverbovyi) [inline tasks/58_missed-letters/missed-letters.js]: try

if(valueToReturn) return valueToReturn

that's it

File: tasks/51_abbreviated-name/abbreviated-name.js [hunk 3]
```diff
@@ -0,0 +1,14 @@
+export function abbreviated(str) {
```
Comment (dverbovyi) [inline tasks/51_abbreviated-name/abbreviated-name.js]: push your own implementation

File: tasks/58_missed-letters/missed-letters.js (hunk 2 above)
Comment (SofiMat) [reply tasks/58_missed-letters/missed-letters.js]: updated

File: tasks/51_abbreviated-name/abbreviated-name.js (hunk 3 above)
Comment (SofiMat) [reply tasks/51_abbreviated-name/abbreviated-name.js]: updated

**Pull request merged**